app/src/main/java/com/kickstarter/libs/rx/transformers/ErrorsTransformerV2.kt

```diff
@@ -0,0 +1,14 @@
+package com.kickstarter.libs.rx.transformers
+
+import io.reactivex.Notification
+import io.reactivex.Observable
+import io.reactivex.ObservableSource
+import io.reactivex.ObservableTransformer
+
+class ErrorsTransformerV2<T> : ObservableTransformer<Notification<T>, Throwable?> {
```

Comment:
Notice that for RXJava 2 instead of `Observable.Transformer` we now implement the interface `ObservableTransformer`